### ISSUE TITLE
Regarding bed file generated

### ISSUE THREAD

**Issue by Proy321** [al-mcintyre/mCaller#38]
Hello Team,

I have successfully extracted the methylation calls and have generated the bed file, but I am trying to find out the differentially methylated regions for the same.Can you suggest me how can I proceed with the bed file generated. It would be really nice if you can provide some insights for the same.

Thanks & Regards
Priyanka Roy

**Comment by al-mcintyre** [comment]
Hi Priyanka, 
I suppose you have generated bed files for different samples between which you're trying to calculate differential methylation? I would recommend in that case analyzing sites detected in both conditions and applying a statistical test on the predicted methylation values (if you have multiple samples per condition, a t test, and if you have only a single sample per condition, Fisher's exact test using the number of reads predicted to be methylated and unmethylated for each condition).

**Comment by PRIYANKA-22091995** [comment]
Yes @al-mcintyre  I have generated multiple files for different samples which i will be using it to calculate the differential methylation, i have 3 different samples(Control) and 3 different samples(Treatment-same condition). So, i should go for t-test?
If yes, how exactly should i proceed with the t-test.
It would be nice to have your insights, and it will be much appreciated.

Thanks

**Comment by al-mcintyre** [comment]
It would be an unpaired t-test between the values for each condition at each site. However, there are issues with applying t-tests to percentages. See discussion here, for instance: https://stats.stackexchange.com/questions/136311/t-test-on-percentage-values-among-experimental-replicate-values. This may give a decent approximation regardless, but the approach would require further validation. 

A more statistically correct robust approach might be to set up a regression model using the counts for "methylated" and "unmethylated" reads (can calculate from the last two columns of the bed file) with and without a term for the interaction between condition and methylation status and then use a likelihood ratio test to test for significance. I've used this before for immunoprecipitation-sequencing approaches, but have not tested it for nanopore data. For IP-seq data, it is further described under the methods for "Peak change detection and generalized linear models" in this paper: https://www.nature.com/articles/s41598-020-63355-3.
